tftpsync/susemanager-tftpsync-recv/configure-tftpsync.sh

```diff
@@ -109,11 +109,11 @@ default_or_input "TFTP Boot directory" TFTPBOOT "/srv/tftpboot"
 
 default_or_input "SUSE Manager Proxy FQDN" SUMA_PROXY_FQDN $(hostname -f)
 
-default_or_input "SUSE Manager Proxy IP Address" SUMA_PROXY_IP $(host $SUMA_PROXY_FQDN | awk '{ print $4 }' || echo "")
+default_or_input "SUSE Manager Proxy IP Address" SUMA_PROXY_IP $(getent hosts $SUMA_PROXY_FQDN | awk '{ print $1 }' || echo "")
```

Comment:
If I get it right, this is should the IP that is resolved for the host, and that is what I want. 
If admins want to add more IPs or a different one, they can set the IP parameter. This will only affect the default value